rust/kernel/print.rs

```diff
@@ -12,6 +12,19 @@ use core::fmt;
 use crate::bindings;
 use crate::c_types::{c_char, c_void};
 
+pub use bindings::___ratelimit;
+pub use bindings::__printk_ratelimit;
+pub use bindings::ratelimit_state;
```

Comment:
I think it is best to avoid `use`ng things from `binding`s to make it clear to the reader it comes from the C side.